jobs/vsphere_cpi/spec

```diff
@@ -80,9 +80,10 @@ properties:
       -----BEGIN CERTIFICATE-----
       MII...
       -----END CERTIFICATE-----
-  vcenter.vmx_options.disk.enableUUID:
-    description: Enable usage of Disk UUIDs for partition lookups of the bosh agent. This is required for Kubernetes Clusters deployed by Bosh to use the Kubernetes CPI (e.g. to provision Pod Volumes). Adds labels to scsi disk devices so the disk device path (`/dev/sd*`) can be discovered via Linux OS. See https://help.ubuntu.com/community/UsingUUID.
-    example: 1 # Allowed values 0 || 1 || "0" || "1"
+  vcenter.vmx_options:
```

Comment:
This is technically a breaking change, but if a tree falls in a forest and nobody is around to hear it...

Reply:
If a tree falls in the forest, but it wasn't ever actually a tree, did a tree fall in the forest?